playFeature/src/main/java/com/panhuk/playfeature/PlayViewModel.kt

```diff
@@ -124,7 +128,7 @@ class PlayViewModel @Inject constructor(
     timerIsActive = true
   }
 
-  private fun isLastQuestion() {
+  private fun checkIsLastQuestion() {
```

Comment:
I guess from the logic of calling methods it should be either `checkLastQuestion` or `isLastQuestion` but not combined.